ui/apps/platform/cypress/integration/collections/collectionsCrudWorkflow.test.js

```diff
@@ -0,0 +1,150 @@
+import withAuth from '../../helpers/basicAuth';
+import { collections as collectionsApi } from '../../constants/apiEndpoints';
+import { visitCollections } from '../../helpers/collections';
+import { hasFeatureFlag } from '../../helpers/features';
+
+// Cleanup an existing collection via API call
```

Comment:
You keep teaching me new idioms!

If first impression is correct that most or all tests depend on each other:
1. Can you add a comment to make the dependency explicit?
2. What are pro and con to call `tryDeleteCollection` at beginning of test versus to add an explicit test at end to delete created and cloned collections via user interface?

Reply:
1. Sure. Ideally I'd have each test fully independent, but I think being able to test the full workflow here is worthwhile and I'm not sure how to keep the tests independent without a lot of repetition. Having a single spec file as independent is probably good enough in this case.
2. This was actually a recommendation from the Cypress docs so that each test is capable of cleaning up before assertions are made. This covers the cases where we run tests out of order (in development, `it.skip`, `it.only`, etc), or a test fails before its cleanup functions can be called. That said, why not both? It might be beneficial to add a catch all cleanup test at the end to keep data from leaking out of this spec, plus we get the benefit of more actual UI test coverage.

Reply:
Indeed. I admit that I did not analyze carefully to what extent you can `skip` or `only` in this test file.